docs/awsdocs/policy-rds.json

```diff
@@ -261,12 +258,18 @@
         {
             "Sid": "ManageDBSnapshotWithOwnerTagRestriction",
             "Action": [
+                "rds:CreateDBSnapshot",
```

Comment:
@rija I would be interested if you can now cleanly destroy your RDS instance by executing `terraform destroy`. I still had the error message below when deleting my personal live environment:
```
│ Error: deleting RDS DB Instance (rds-server-staging-pli): operation error RDS: DeleteDBInstance, https response error StatusCode: 403, RequestID: 8e321d91-f359-4917-8ddc-33e6324e6658, api error AccessDenied: User: arn:aws:iam::049839813732:user/pli is not authorized to perform: rds:CreateDBSnapshot on resource: arn:aws:rds:ap-south-1:049839813732:snapshot:snapshot-final-live-pli-20241212091002-rds-server-staging-pli-92c89eda because no identity-based policy allows the rds:CreateDBSnapshot action
```